src/js/components/ExtraInfo/index.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import React from 'react';
+import { views } from '../../helpers';
+
+type Props = {
+  data: string,
+  view: string
+}
+
+function showInfo( props: Props ) {
+  const { view, data } = props;
+  return view === views.POSTER ?
+    <div className='extra-info__content'>Films by {data} genre</div> :
+    <div className='extra-info__content'>{data} movies found</div>;
+}
+
+const ExtraInfo = (props: Props) => (
+  <div className='extra-info'>
+    {showInfo(props)}
```

Comment:
why not just:
```jsx
<div className='extra-info'>
   <div className='extra-info__content'>{getMessage(view, data)}</div>
```
?
Since wrapper for the text is the same